src/commands/moderation/commandTable.ts

```diff
@@ -0,0 +1,117 @@
+import { ApplicationCommandOptionType, ApplicationCommandType, CommandInteraction } from 'discord.js';
+import { S3Client, PutObjectCommand } from '@aws-sdk/client-s3';
+import { slashCommand, slashCommandStructure, Logger, constantsConfig } from '../../lib';
+
+const data = slashCommandStructure({
+    name: 'generate_command_table',
```

Comment:
Could we make that with `-` instead of `_`, it feels more natural as a command
```suggestion
    name: 'generate-command-table',
```

Reply:
Potentially we did - has not happened yet if so!

I can merge this one as is, and I'll go through and refactor all at once in the next PR?

Reply:
sounds good :)